modules/virt-creating-secondary-localnet-udn.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-secondary-udn.adoc        
+
+:_mod-docs-content-type: PROCEDURE                                  
+[id="virt-creating-secondary-localnet-udn_{context}"]                                  
+= Creating a user-defined-network for localnet topology by using the CLI
+
+You can create a secondary cluster-scoped user-defined-network (CUDN) for localnet topology by using the CLI.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the Kubernetes NMState Operator.
+
+.Procedure
+. Create a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: mapping # <1>
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' # <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: tenantblue # <3>
+        bridge: br-ex # <4>
+        state: present # <5>
+----
+<1> The name of the configuration object.
+<2> Specifies the nodes to which the node network configuration policy is to be applied. The recommended node selector value is `node-role.kubernetes.io/worker: ''`.
+<3> The name of the additional network from which traffic is forwarded to the OVS bridge. This attribute must match the value of the `spec.config.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
+<4> The name of the OVS bridge on the node. This value is required if the `state` attribute is `present` or not specified.
+<5> The state of the mapping. Must be either `present` to add the mapping or `absent` to remove the mapping. The default value is `present`.
++
+[NOTE]
+====
+{VirtProductName} does not support Linux bridge bonding modes 0, 5, and 6. For more information, see link:https://access.redhat.com/solutions/67546[Which bonding modes work when used with a bridge that virtual machine guests or containers connect to?].
+====
+
```

Comment:
Do we need to apply the NodeNetworkConfigurationPolicy object here?

Reply:
Done